### DISCUSSION TITLE
Creating prisma model dynamically

### DISCUSSION THREAD

**Discussion by sPredictorX1708** [prisma/prisma#11424] [Q&A]
I am looking for creating models dynamically. How can I do it using prisma apart from running nodejs command?

**Comment by austincrim** [comment]
Hey @dubesar 👋 ,

This is not currently possible. It is possible to modify your database schema at runtime using [`$executeRaw`](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#executeraw) and SQL, but this is usually not recommended.

If you have valid use cases and ideas about how we could do this, please file a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=).
Thanks!

**Reply by vladi-stevanovic** [reply]
👋 Hello @dubesar I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! :raised_hands:

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
